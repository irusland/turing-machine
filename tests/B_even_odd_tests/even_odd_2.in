3
even odd stop
even stop
6
even 0 -> even _ R
even 1 -> odd _ R
even _ -> stop 0 S
odd 0 -> odd _ R
odd 1 -> even _ R
odd _ -> stop 1 S
10100