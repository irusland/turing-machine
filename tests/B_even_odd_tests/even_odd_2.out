STOP after 6 transitions
0
stop 0
